dsls/sql/src/test/java/org/apache/beam/dsls/sql/planner/MockedBeamSqlTable.java

```diff
@@ -119,10 +117,11 @@ public RelDataType apply(RelDataTypeFactory a0) {
 
   @Override
   public BeamIOType getSourceType() {
-    return BeamIOType.UNBOUNDED;
+    return BeamIOType.BOUNDED;
```

Comment:
I intended to make this change in another PR(to avoid unrelated changes). Created a separate issue to track this: https://issues.apache.org/jira/browse/BEAM-2515